File: Examples/VisualBasic/ModifyingAndConvertingImages/DNG/ConvertingDNGToJPG.vb


Namespace Aspose.Imaging.Examples.VisualBasic.ModifyingAndConvertingImages.DNG

    Public Class ConvertingDNGToJPG

        Public Shared Sub Run()

            ' To get proper output please apply a valid Aspose.Imaging License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx."

            ' ExStart: SyncRootProperty

            'Declare variables to store file paths for input and output images.
            Dim sourceFiles As String = "Path_to_source_folder\Source\HDR - 3c.dng"
            Dim destPath As String = "Path_to_results_folder\Results\result.jpg"

            ' Create an instance of Image class and load an exiting DNG file.
            ' Convert the image to DngImage object.
            Using image1 As Aspose.Imaging.FileFormats.Dng.DngImage = DirectCast(Image.Load(sourceFiles), Aspose.Imaging.FileFormats.Dng.DngImage)

                ' Create an instance of JpegOptions class.
                ' convert and save to disk in Jpeg file format.
                image1.Save(destPath, New Aspose.Imaging.ImageOptions.JpegOptions())
            End Using

            ' ExEnd: SyncRootProperty


        End Sub

    End Class

End Namespace

File: Examples/CSharp/ModifyingAndConvertingImages/DNG/ConvertingDNGToJPG.cs
// GIST-ID: 66c9eecc56a847b286215fc22d1a8ac0
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Imaging.Examples.CSharp.ModifyingAndConvertingImages.DNG
{
    class ConvertingDNGToJPG
    {
        public static void Run()
        {
            // To get proper output, please apply a valid Aspose.Imaging license. You can purchase a full license
            // or get a 30‑day temporary license from https://www.aspose.com/purchase/default.aspx.

            // ExStart: SyncRootProperty

            // Declare variables to store file paths for input and output images.
            string sourceFiles = "Path_to_source_folder\\Source\\HDR - 3c.dng";
            string destPath = "Path_to_results_folder\\Results\\result.jpg";

            // Create an instance of the Image class and load an existing DNG file.
            // Convert the image to a DngImage object.
            using (Aspose.Imaging.FileFormats.Dng.DngImage image = (Aspose.Imaging.FileFormats.Dng.DngImage)Image.Load(sourceFiles))
            {
                // Create an instance of the JpegOptions class.
                // Convert and save the image to disk in JPEG format.
                image.Save(destPath, new Aspose.Imaging.ImageOptions.JpegOptions());
            }

            // ExEnd: SyncRootProperty
        }
    }
}